src/web/aws-cloudwatch/StepAuthorize.jsx

```diff
@@ -23,57 +24,60 @@ const StepAuthorize = ({ onChange, onSubmit }) => {
           <DisappearingInput id="password" type="password" />
           {/* Continue on, Nothing to See Here */}
 
-          <Input id="awsCloudWatchName"
-                 type="text"
-                 value={formData.awsCloudWatchName ? formData.awsCloudWatchName.value : ''}
-                 onChange={onChange}
-                 placeholder="CloudWatch Integration Name"
-                 label="Name of integration"
-                 autoComplete="off"
-                 required />
+          <ValidatedInput id="awsCloudWatchName"
+                          type="text"
+                          fieldData={formData.awsCloudWatchName}
+                          onChange={onChange}
+                          placeholder="CloudWatch Integration Name"
+                          label="Name of integration"
+                          autoComplete="off"
+                          required />
 
-          <Input id="awsCloudWatchDescription"
-                 type="textarea"
-                 label="Integration description"
-                 placeholder="CloudWatch Integration Description"
-                 onChange={onChange}
-                 value={formData.awsCloudWatchDescription ? formData.awsCloudWatchDescription.value : ''}
-                 rows={4} />
+          <ValidatedInput id="awsCloudWatchDescription"
+                          type="textarea"
+                          label="Integration description"
+                          placeholder="CloudWatch Integration Description"
+                          onChange={onChange}
+                          fieldData={formData.awsCloudWatchDescription}
+                          rows={4} />
 
-          <Input id="awsCloudWatchAwsKey"
-                 type="password"
-                 label="AWS Key"
-                 placeholder="CloudWatch Integration AWS Key"
-                 onChange={onChange}
-                 value={formData.awsCloudWatchAwsKey ? formData.awsCloudWatchAwsKey.value : ''}
-                 autoComplete="off"
-                 pattern="AK[A-Z0-9]{18}"
-                 minLength="20"
-                 required />
+          <ValidatedInput id="awsCloudWatchAwsKey"
+                          type="password"
```

Comment:
I believe that the AWS Key field should be unmasked. This is how it shows in the AWS console. This helps the user to confirm they are pasting the correct credential set. The AWS Secret should continue to be masked.

Reply:
> I believe that the AWS Key field should be unmasked.

Should we unmask the Review page as well?